modernism/static/_js/modernism.js

```diff
@@ -126,6 +126,24 @@ const setModalImage = (direction) => {
 
 }
 
+//Get the button
+let mybutton = document.getElementById("myBtn");
+// When scroll down 100px from the top of the document, show the button
```

Comment:
Please remove comments here, as it is clear by reading the code what happens.

Reply:
I will remove it.